### PR TITLE
Swifter's Ship Emporium But Updated

### PR DESCRIPTION
**Pull request by triplezeta**
title

## Changelog
:cl: swifterjack
add: ships
/:cl:

### CODE REVIEW

**Comment by Firefox13** [comment]
Chief Medical Officer on Spitfire lacks access to advanced chemical closet on that ship for some reason. Not really a problem given they get emag but eh

File: _maps/configs/raven.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+	"map_name": "Raven-class Heavy Transporter",
+	"map_short_name": "Raven-class",
+	"map_path": "_maps/shuttles/shiptest/nanotrasen_raven.dmm",
+	"map_id": "nanotrasen_raven",
+	"job_slots": {
+		"Captain": 1,
+		"Head of Security": 1,
+		"Chief Medical Officer": 1,
+		"Head Of Personnel": 1,
```
Comment (MarkSuckerberg) [inline _maps/configs/raven.json]:
```suggestion
		"Head of Personnel": 1,
```
can't believe I made this case-sensitive

File: _maps/configs/statio.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	"map_path": "_maps/shuttles/shiptest/nanotrasen_station.dmm",
+	"map_id": "nanotrasen_station",
+	"job_slots": {
+		"Captain": 1,
+		"Head of Personnel": 1,
+		"Chief Engineer": 1,
+		"Chief Medical Officer": 1,
+		"Research Director": 1,
+		"Head of Security": 1,
+		"Solgov Representative": 1,
```
Comment (MarkSuckerberg) [inline _maps/configs/statio.json]:
```suggestion
		"SolGov Representative": 1,
```
suffering

**Review by MarkSuckerberg — CHANGES_REQUESTED** [review]
these'll get propogated to the PR on swifter's fork so I'm doing it here coz I'm lazy

**Pull request merged**